resip/stack/UdpTransport.cxx

```diff
@@ -321,72 +316,43 @@ UdpTransport::processTxOne(SendData *data)
 void
```

Comment:
Note:  There are references to the removed KEEP_BUFFER flag in the above comment, we need to clean up.